app/src/main/java/com/endava/myendava/fragments/ProfileFragment.java

```diff
@@ -137,22 +128,14 @@ private void setViews() {
         mRecyclerView.setHasFixedSize(true);
         RecyclerView.LayoutManager layoutManager = new LinearLayoutManager(getContext());
         mRecyclerView.setLayoutManager(layoutManager);
-        adapter = new ChipsAdapter(getContext(), new Profile(),
-                tagsMap, this);
+        adapter = new ChipsAdapter(getContext(), mSharedPreferences.getUserEmail(),false, new Profile(),
+                tagsMap, this, this);
         mRecyclerView.setAdapter(adapter);
 
-        if (!getArguments().getBoolean(ARG_IS_USER_PROFILE, true)) {
-            ((AppCompatActivity) getActivity()).setSupportActionBar(mToolbar);
-            ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
-            if (actionBar != null) {
-                actionBar.setDisplayHomeAsUpEnabled(true);
-            }
-        }
+        if (!getArguments().getString(ARG_EMAIL, "").equals(EmailType.OWN_EMAIL.getType())) {
```

Comment:
this if is empty ... either we need it or we don't need it

Reply:
Now we dont need it, we needed before, when we hide the FloatingButton for other users.